ui/src/main/resources/org/pentaho/di/ui/spoon/messages/messages_de_DE.properties

```diff
@@ -5,7 +5,7 @@ Spoon.Application.Name=Spoon
 Spoon.Menu.File=&Datei
 Spoon.Menu.File.New=&Neu
 Spoon.Menu.File.Open=\uFFFD&ffnen
-Spoon.Menu.File.OpenVFS=Datei von &URL \uFFFDffnen
+Spoon.Menu.File.OpenVFS=Datei von &URL \uFFFDffnen...
```

Comment:
This is one of the changes not asked for in the JIRA case. Not sure if this option should have ellipsis.

Reply:
@pentaho-lmartins, the JIRA case was created in 2016 relating to version (as old as) 6.1.0.0 GA; since then, we had several versions and new options appeared on our menus.
I had a look and found several entries that were missing an ellipsis at the end (one or two on every languages and some in only some languages).

Reply:
I left a comment on the issue explaining what was altered besides what is asked for: https://jira.pentaho.com/browse/PDI-15730?focusedCommentId=378317&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-378317